lib/kubeclient/inflections.rb

```diff
@@ -0,0 +1,4 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.irregular 'resource_quota', 'resource_quotas'
+  inflect.irregular 'resourcequota', 'resourcequotas'
+end
```

Comment:
> Yeah, if there's a way you can do that without uglifying your code too much, that's probably a more "pure" choice, for a library.

oh it surely will uglify :)
because it must add an "if"